packages/gateway/src/utils/json-response.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-env serviceworker */
+export class JSONResponse extends Response {
+  /**
+   * @param {BodyInit} body
+   * @param {ResponseInit} [init]
+   */
+  constructor(body, init = {}) {
+    init.headers = init.headers || {}
+    init.headers['Content-Type'] = 'application/json;charset=UTF-8'
+    super(JSON.stringify(body), init)
+  }
+}
+
+export function notFound(message = 'Not Found') {
+  return new JSONResponse({ message }, { status: 404 })
+}
```

Comment:
I think we don't need JSON responses.

Reply:
Maybe for CID status, but removing until needed